docs/commands.md

```diff
@@ -0,0 +1,132 @@
+    the globalConfig file in the parent directory of the source provided.
+    Only *.json* and *.yaml* files are accepted.
+* `--ta-version` - [optional] override current version of TA. The default
+    version is version specified in `globalConfig.json` or `globalConfig.yaml`.
+    A Splunkbase compatible version of SEMVER is used by default.
+* `-o` / `--output` - [optional] output folder to store the build add-on.
+   By default, it is saved in the `current directory/output` folder.
+    Absolute paths are accepted as well.
+* `--python-binary-name` - [optional] Python binary name to use when
+    installing Python libraries. The default is `python3`.
+* `-v` / `--verbose` - [optional] shows detailed information about
+    created/copied/modified/conflict files after build is complete.
+    This option is in the experimental mode. The default is `False`.
+* `--pip-version` - [optional] pip version that is used to install python libraries. The default is `latest`.
+* `--pip-legacy-resolver` - [optional] Use old pip dependency resolver by adding flag '--use-deprecated=legacy-resolver'
+    to pip install command. The default is`False`. 
+    >**NOTE:** This flag is deprecated and will be removed from pip in the future.
+    Instead of using this flag, the correct solution would be to fix the packages your project depends on to work properly with the new resolver. Additionally, this flag is not compatible with pip version `23.2`. Use `23.2.1` instead.
+* `--ui-source-map` - [optional] if present generates front-end source maps (.js.map files), that helps with code debugging.
+
+### Verbose mode
+
+The verbose mode is available for `v5.35.0` and later.
+
+Running `ucc-gen build -v` or `ucc-gen build --verbose` prints additional information about
+what was exactly created / copied / modified / conflicted after the build is complete. It does
+not scan the `lib` folder due to the nature of the folder.
+
+See the following description of what each state means:
+
+* `created`: the file is not in the original package and was created during the build process.
+* `copied`: the file is in the original package and was copied during the build process.
+* `modified`: the file is in the original package and was modified during the build process.
+* `conflict`: the file is in the original package and was copied during the build process, but may be generated by UCC itself, so incorrect usage can stop the add-on from working.
+
+## `ucc-gen init`
+
+`ucc-gen init` initializes the add-on. This is available on `v5.19.0` and later.
+The `ucc-gen init` command initializes the add-on and bootstraps some code in the
+modular input which you, as a developer, can extend for your needs.
+
+Apart from standard files needed for the add-on, it also adds search head
+clustering files in the `default/server.conf` file and reload triggers in the
+`default/app.conf` file. Those files will be soon <!--- verb is missing ---> by generated automatically by the
```

Comment:
Those files will be soon <!--- verb is missing ---> by generated automatically by the `ucc-gen build` command itself. @artemrys what will happen the files?